### PR TITLE
GH-173 - Assign modal fix

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
GH-173 - Assign modal fix

#### Ticket Link
#173

### CODE REVIEW

File: webapp/src/components/user_selector/autocomplete_selector.tsx [hunk 1]
```diff
@@ -123,6 +124,7 @@ export default function AutocompleteSelector(props: Props) {
             {labelContent}
             <div className={inputClassName}>
                 <AsyncSelect
+                    autoFocus={props.autoFocus || false}
```
Comment (mickmister) [inline webapp/src/components/user_selector/autocomplete_selector.tsx:127]:
```suggestion
                    autoFocus={Boolean(props.autoFocus)}
```

**Review by mickmister — APPROVED** [review]
LGTM :+1:  Just one suggestion for improvement

File: webapp/src/widget/menuItem.tsx [hunk 2]
```diff
@@ -20,8 +20,9 @@ const MenuItem = (props: Props) => {
     const {icon, shortcut, action, text} = props;
 
     useEffect(() => {
-        function handleKeypress(e: {key: string}) {
+        function handleKeypress(e: any) {
```
Comment (larkox) [inline webapp/src/widget/menuItem.tsx:23]:
```suggestion
        function handleKeypress(e: KeyboardEvent) {
```

**Review by larkox — APPROVED** [review]
LGTM, just a minor change.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
The focus moves to the assign modal as expected and keystrokes begin to search through the user list as expected. 

Thanks @asaadmahmood 🎉  
**note**: To clarify I was no longer able to reproduce the issue where I can action the shortcut keys with the assign modal open. But in any case - I did test again with this PR and don't see the issue.

**Pull request merged**